data/assets/scene/solarsystem/heliosphere/2012/sun_earth_2012_fieldlines_batsrus.asset

```diff
@@ -0,0 +1,169 @@
+local transforms = asset.require('scene/solarsystem/planets/earth/magnetosphere/transforms_magnetosphere.asset')
+
+-- Specifying transfer functions
+local transferFunctions = asset.syncedResource({
+  Name = "Fieldlines Transfer Functions",
+  Type = "HttpSynchronization",
+  Identifier = "sun_earth_event_july_2012-fieldlines_transferfunctions",
+  Version = 1
+})
+
+-- Specifying transfer functions
```

Comment:
Seems like this is data, and not a transfer function? Both of these comments can be removed anyway imo. ot really needed :)